packages/medusa/src/models/line-item.ts

```diff
@@ -283,6 +320,15 @@ export class LineItem extends BaseEntity {
  *   includes_tax:
  *     description: "[EXPERIMENTAL] Indicates if the line item unit_price include tax"
  *     type: boolean
+ *   original_item_id:
+ *     description: "[EXPERIMENTAL] The id of the original line item"
```

Comment:
```suggestion
 *     description: "[EXPERIMENTAL] The id of the original line item before the order edit"
```
or something along those lines just to mention "order edit"

Reply:
The original item id can be used for other purposes than the order edit.